app/components/AdminDropdown/AgencyDropdown.tsx

```diff
@@ -0,0 +1,46 @@
+import { Agency } from "~/gen";
+import { AdminDropdownProps, AdminDropdown } from ".";
+import { ManagingAgencyAcronym } from "../../utils/types";
+import { analytics } from "../../utils/analytics";
+
+export interface AgencyDropdownProps
+  extends Pick<AdminDropdownProps, "selectValue"> {
+  agencies: Array<Agency> | null;
+  onSelectValueChange?: (value: null | string) => void;
+}
+export function AgencyDropdown({
+  selectValue,
+  agencies,
+  onSelectValueChange = () => null,
+}: AgencyDropdownProps) {
+  const updateManagingAgencyAcronym = (
+    nextManagingAgencyAcronym: ManagingAgencyAcronym,
+  ) => {
+    analytics({
+      category: "Dropdown Menu",
+      action: "Change Managing Agency",
+      name: nextManagingAgencyAcronym,
+    });
+
+    onSelectValueChange(nextManagingAgencyAcronym);
+  };
+
+  const agencyOptions = agencies
```

Comment:
I'll make tickets so that we're sorting this in the API and the frontend can serve them as-is